src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -133,6 +133,46 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
   }
 }
 
+/**
+ * Uses Drupal content via a new GraphQL query or the cached result of a
+ * previous query. This is where the cache is saved.
+ *
+ * @param {Object} buildOptions
+ * @return {Object} - The result of the GraphQL query
+ */
+async function getSideNavsViaGraphQL(buildOptions = global.buildOptions) {
+  global.buildtype = buildOptions.buildtype;
+  let sideNavs;
+
+  const sideNavFile = path.join(
+    buildOptions.cacheDirectory,
+    'drupal',
+    'side-nav-menus.json',
+  );
+
+  if (shouldPullDrupal(buildOptions)) {
+    const contentApi = getApiClient(buildOptions);
+    log('Pulling side nav menus from Drupal...');
+    await contentApi.getSideNavigations().then(res => {
+      sideNavs = res.data.sideNavMenus;
+
+      // Write them to .cache/{buildtype}/drupal/side-nav-menus.json
+      fs.ensureDirSync(buildOptions.cacheDirectory);
+      fs.emptyDirSync(path.dirname(sideNavFile));
+      fs.writeJsonSync(sideNavFile, sideNavs, { spaces: 2 });
+    });
```

Comment:
Since we're already using `await`, maybe for consistency:

```suggestion
    const response = await contentApi.getSideNavigations();
    sideNavs = response.data.sideNavMenus;

    // Write them to .cache/{buildtype}/drupal/side-nav-menus.json
    fs.ensureDirSync(buildOptions.cacheDirectory);
    fs.emptyDirSync(path.dirname(sideNavFile));
    fs.writeJsonSync(sideNavFile, sideNavs, { spaces: 2 });
```